skyportal/handlers/api/followup_request.py

```diff
@@ -808,6 +809,36 @@ def delete(self, request_id):
             return self.success()
 
 
+class TargetOfOpportunityConstraint(Constraint):
+    """
+    Prioritize target of opportunity targets earlier.
```

Comment:
This isn't very clear at least for me. The comment doesn't really help me understand what the method is supposed to do.

Reply:
Yes added better text.